parsec/profiling.h

```diff
@@ -85,26 +85,34 @@
 /**
  * @brief  Flag used when an info object is attached to the event
  */
-#define PARSEC_PROFILING_EVENT_HAS_INFO     (1<<0)
+#define PARSEC_PROFILING_EVENT_HAS_INFO         (1<<0)
 /**
  * @brief Flag used when the event is a reschedule of a previous event
  */
-#define PARSEC_PROFILING_EVENT_RESCHEDULED  (1<<1)
+#define PARSEC_PROFILING_EVENT_RESCHEDULED      (1<<1)
 /**
  * @brief Flag used when the event's info is a counter
  * @details The event's info (if present) is an integer that
  *          should be accumulated to a value starting at 0.
  *          This might be useful to represent countable entities,
  *          like amount of tasks pending, amount of memory allocated,
  *          etc. */
-#define PARSEC_PROFILING_EVENT_COUNTER      (1<<2)
+#define PARSEC_PROFILING_EVENT_COUNTER          (1<<2)
 /**
  * @brief Constant to use when no handle/object ID can be associated
  *        with an event
  * @details Some functions take a handle/object ID as a parameter.
  *          When such ID is not applicable, this constant should be
  *          passed to the parameter.
  */
+#define PARSEC_PROFILING_EVENT_TIME_AT_START    (1<<3)
```

Comment:
You should also define a PARSEC_PROFILING_EVENT_TIME_AT_END as 0 and use it symmetrically in the pins profiler that you modify, just for the clarity of reading.